sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -126,7 +126,10 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
       `IdentityClient: refreshing access token with client ID: ${clientId}, scopes: ${scopes} started`
     );
 
-    const { span, updatedOptions: newOptions } = createSpan("IdentityClient-refreshAccessToken", options);
+    const { span, updatedOptions: newOptions } = createSpan(
+      "IdentityClient-refreshAccessToken",
+      options
+    );
```

Comment:
I ran rushx format and it changed a bunch of files. I rather have these formatting changes than not have them. It will make it easier to do further hotfixes if necessary.